backend/segmentation/ml/model_loader.py

```
@@ -1271,6 +1254,110 @@ def predict_microcapsule(self, image: Image.Image, threshold: float = 0.5,
             self.current_model = None
             self.release_model('microcapsule')
 
+    def predict_disintegration(self, image: Image.Image, threshold: float = 0.5,
+                               detect_holes: bool = True,
+                               timeout: Optional[float] = None) -> Dict[str, Any]:
+        """Run the spheroid-disintegration model (UNet++/EffB5, 3-class).
+
+        The model predicts 0 = background, 1 = corona, 2 = core. The foreground
+        (corona ∪ core) becomes the outer spheroid polygons; the core (class 2)
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `∪` (UNION). Did you mean `U` (LATIN CAPITAL LETTER U)?

(RUF002)